switchboard/scripts/README.md

```diff
@@ -0,0 +1,3 @@
+Directory contents:
+
+- `ed25519-genkeypair.sh` - used to generated ed25519 keypairs for supervisor authentication
```

Comment:
I don't know whether we need this README at all. If all the scripts have a help message prompt or a comment at the top of the respective file, I think that's enough docs for now.